save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/AgentDownloadUtils.kt

```diff
@@ -0,0 +1,78 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+/**
+ * Utils for building curl command that downloads save-agent and save-demo-agent to pod
+ */
+
+package com.saveourtool.save.utils
+
+import org.intellij.lang.annotations.Language
+
+@Language("bash")
+private val defaultShellOptions: Array<out String> = arrayOf(
```

Comment:
may be sequence? anyway you do such cast in `getShellOptions`

Reply:
Yep, will do it

Reply:
Changed `Array<out String>` to `Sequence<String>`